src/globalsAugment.d.ts

```diff
@@ -0,0 +1,18 @@
+import BigNumber from 'bignumber.js';
```

Comment:
Add a comment at the top of this file about how it is a hack in order to import the `bignumber.js` definitions from the module within a global definition.